README.md

```diff
@@ -31,10 +32,10 @@ Templates:
 * [Decision record template for Alexandrian pattern](locales/en/templates/decision-record-template-for-alexandrian-pattern/)
 * [Decision record template for business case](locales/en/templates/decision-record-template-for-business-case/)
 * [Decision record template of the MADR project](locales/en/templates/decision-record-template-of-the-madr-project/)
-* [Decision record template using Planguage](locales/en/templates/decision-record-template-using-planguage/)  
```

Comment:
some trailing white spaces, include this, are removed, by my local editor's linter.
split commits for this kind of changes in next PRs.